Python/lgmpy/Lgm_Vector.py

```diff
@@ -241,13 +246,39 @@ def __div__(self, other):
 
         @version: V1: 22-Dec-2010 (BAL)
         """
-        if isinstance(other, (int, float, long)):
+        if isinstance(other, self._typetup):
             x = self.x / other
             y = self.y / other
             z = self.z / other
             return Lgm_Vector(x, y, z)
         else:
-            raise(ArithmeticError("Cannot subtract %s from a Lgm_Vector" % (type(other))))
+            raise ArithmeticError("Cannot divide an Lgm_Vector by type %s" % (type(other)))
+
+    __truediv__ = __div__
+
+    def __floordiv__(self, other):
+        """
+        divide a scalar into each component
+
+        @param other: scalar to divide into vector
+        @type other: (int, long, float)
+
+        @return: new vector
+        @rtype: Lgm_Vector
+
+        @author: Brian Larsen
+        @organization: LANL
+        @contact: balarsen@lanl.gov
+
+        @version: V1: 22-Dec-2010 (BAL)
+        """
```

Comment:
(I had a crash so this might be a duplicate comment on this, in which case the other one is better.)

The docstring should probably be updated (for author and the fact that it's floor div). What's the method resolution order on Py2k and Py3k? Will 2k always do `div` for `/` and `floordiv` for `//`, and 3k do `truediv` for `/` and `floordiv` for `//`? Just wondering about the assignment and if it's the right way around. I'm also going to suggest a future import for division elsewhere; that might affect this.

Reply:
See previous. Marking as resolved since the comment is a duplicate.